homeassistant/components/fritz/sensor.py

```diff
@@ -86,55 +87,63 @@ class SensorData(TypedDict):
     "external_ip": SensorData(
         name="External IP",
         device_class=None,
+        state_class=None,
         unit_of_measurement=None,
         icon="mdi:earth",
         state_provider=_retrieve_external_ip_state,
     ),
     "uptime": SensorData(
         name="Uptime",
         device_class=DEVICE_CLASS_TIMESTAMP,
+        state_class=None,
         unit_of_measurement=None,
         icon=None,
         state_provider=_retrieve_uptime_state,
     ),
     "kib_s_sent": SensorData(
         name="KiB/s sent",
         device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
         unit_of_measurement="KiB/s",
         icon="mdi:web",
         state_provider=_retrieve_kib_s_sent_state,
     ),
     "kib_s_received": SensorData(
         name="KiB/s received",
         device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
         unit_of_measurement="KiB/s",
         icon="mdi:web",
         state_provider=_retrieve_kib_s_received_state,
     ),
     "max_kib_s_sent": SensorData(
         name="Max KiB/s sent",
         device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
         unit_of_measurement="KiB/s",
         icon="mdi:web",
         state_provider=_retrieve_max_kib_s_sent_state,
     ),
     "max_kib_s_received": SensorData(
         name="Max KiB/s received",
         device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
         unit_of_measurement="KiB/s",
         icon="mdi:web",
         state_provider=_retrieve_max_kib_s_received_state,
     ),
     "mb_sent": SensorData(
         name="MB sent",
         device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
This is a counter right? (provides a sum?)

In that case `last_reset` must be provided as well.

Reply:
As discussed on discord.

```suggestion
```